net.sourceforge.quakespasm.Quakespasm.appdata.xml

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<application>
+  <id type="desktop">net.sourceforge.quakespasm.Quakespasm</id>
+  <name>Quakespasm</name>
+  <developer_name>Quakespasm contributors</developer_name>
+  <summary>A modern, *nix-friendly source port of the original Quake engine</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-2.0-only</project_license>
+  <url type="homepage">https://quakespasm.sourceforge.net/</url>
+  <description>
+    <p>Quakespasm is a *Nix friendly Quake Engine based on the SDL port of the popular FitzQuake.
+      It includes some new features, important fixes, and aims for portability and 64 bit correctness.</p>
+    <p>Features:
+      <ul>
+        <li>Supports Windows, Mac OS X, Linux, other unices.</li>
+        <li>Well tested on both 32 bit and 64 bit platforms.</li>
+        <li>Well tested on both little endian and big endian platforms.</li>
+        <li>OpenGL improvements. In-game resolution changing.</li>
+        <li>Widescreen FOV (field of view) support and automatic Hor+ FOV scaling.</li>
+        <li>Improved sound support. Support for OGG, MP3, Opus and WAV external music files.</li>
+        <li>Improved game controller support using SDL2.</li>
+        <li>Countless bug fixes, security fixes and other improvements.</li>
+        <li>Support for playing the 'Quake 2021 re-release' content</li>
+      </ul></p>
+    <p>Quakespasm requires a copy Quake game data, at the least in the form of the shareware data file
+      <span style="font-family: monospace">pak0.pak</span>. In order to set this up, create the directory
+      <span style="font-family: monospace">~/.config/quakespasm/data</span> on your computer and copy the
+      <span style="font-family: monospace">id1</span> directory from a copy of Quake there. Quakespasm
```

Comment:
Not supported https://freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-description